packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { disabled, theme } = props;
+  const { semanticColors } = theme;
+
+  return {
+    root: [
+      'ms-Link',
+      getFocusStyle(theme),
+      {
+        color: semanticColors.link,
+        selectors: {
+          'button&': {
```

Comment:
Since the component has a check for an 'href' attribute that decides whether the element renders and and anchor or button, why don't we put that in a boolean that can be passed to styles as a prop. In that case, we don't have to specify the rendered element ( button& or a& ) and instead can style just style based on the boolean prop.

Reply:
Good idea. I've refactored so that the styles are applied based on the props based in, without any HTML selectors in the CSS.